types/console-log-level/index.d.ts

```diff
@@ -0,0 +1,20 @@
+// Type definitions for console-log-level 1.4
+// Project: https://github.com/watson/console-log-level
+// Definitions by: Ali Ijaz Sheikh <https://github.com/ofrobots>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.1
+
+type PrefixFunction = (level: string) => string;
+
+type LogLevelNames = "trace" | "debug" | "info" | "warn" | "error" | "fatal";
```

Comment:
Is there a way to export `LogLevelNames` given that we are doing a CommonJS style export below?

Reply:
```
namespace consoleLogLevel {
  // anything in here will get exported as long as the namespace name matches the exported function's name
}
```